provider/aws/fixtures/self-signed-ca.crt

```diff
@@ -0,0 +1,22 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
Is it possible to reuse this certs or location https://github.com/kubernetes-sigs/external-dns/tree/master/internal/testresources?